packages/gatsby/src/commands/graphiql.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+
+const express = require(`express`)
+const graphqlHTTP = require(`express-graphql`)
+const { store } = require(`../redux`)
+const bootstrap = require(`../bootstrap`)
+
+module.exports = async (program: any) => {
+  let { port, host } = program
+  port = typeof port === `string` ? parseInt(port, 10) : port
```

Comment:
maybe we can move this into the yargs option? just go with a int for the type instead of string

Reply:
👍 I'll update this.

Reply:
@jquense I guess this involves changing [the type in create-cli.js](https://github.com/m-allanson/gatsby/blob/8224f0d95da24620363499aaf91a8d731064af77/packages/gatsby-cli/src/create-cli.js#L162) to `number`? 

Which would then be inconsistent with the `serve` and `develop` commands. I'm a bit wary of changing the types for the other commands without any tests in place...